docs/commerce/checkout/customize_checkout.md

```diff
@@ -0,0 +1,202 @@
+---
+description: Customize the existing checkout functionality to support additional functions.
+edition: commerce
+---
+
+# Customize checkout
+
+When you work with your Commerce implementation, you can review and modify 
+the checkout configuration.
+
+Checkout is an essential component of the Commerce offering.
+It collects data that is necessary to create an order, including:
+
+- payment method
+- shipping method
+- billing / delivery address
+
+It could also collect any other information that you find necessary.
+
+Depending on your needs, the checkout process can be either complicated or very simple. 
+For example, if the website is selling airline tickets, you may need several additional steps 
+with passengers defining their special needs.
+On the other side of the spectrum would be a store that sells books with personal pickup, 
+where one-page checkout would be enough.
```

Comment:
Maybe link to the two heading below, to make it clear that the page describes two separate procedures?